dev-scheme/chicken/chicken-4.13.0.recipe

```diff
@@ -0,0 +1,122 @@
+SUMMARY="A Scheme interpreter and native Scheme to C compiler"
+DESCRIPTION="CHICKEN is a compiler for the Scheme programming language. \
+CHICKEN produces portable and efficient C, supports almost all of the R5RS \
+Scheme language standard, and includes many enhancements and extensions. \
+CHICKEN runs on Linux, MacOS X, Windows, and many Unix flavours."
+HOMEPAGE="http://www.call-cc.org/"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2008-2017 The Chicken Team
+	2000-2007 Felix L. Winkelmann"
```

Comment:
Oops, you only addressed part of my comment in https://github.com/haikuports/haikuports/pull/2699#discussion_r195930962.
The recommended sorting is not `REVISION`+`COPYRIGHT`+`LICENSE`,  but `COPYRIGHT`+`LICENSE`+`REVISION`.

That said, I'm OK with bending the rules a little bit if you wish ;)

Reply:
No problem, will sort it out in a bit :)